addon/components/frost-select-dropdown.js

```diff
@@ -47,6 +47,7 @@ export default Component.extend({
     onSelect: PropTypes.func.isRequired,
     receivedHook: PropTypes.string,
     selectedItems: PropTypes.arrayOf(PropTypes.object),
+    wrapOptionText: PropTypes.bool,
```

Comment:
Thoughts on just making this `wrapLabels`? I find the name `wrapOptionText` to be a mouthful and a bit confusing.